fhir-parent/pom.xml

```diff
@@ -62,8 +62,8 @@
         <fhir-persistence-jdbc.index>MINIMAL_JSON</fhir-persistence-jdbc.index>
         <fhir-server-test.index>MINIMAL_JSON</fhir-server-test.index>
         <java.version>1.8</java.version>
-        <fhir-examples.version>4.7.0-SNAPSHOT</fhir-examples.version>
-        <fhir-tools.version>4.7.0-SNAPSHOT</fhir-tools.version>
+        <fhir-examples.version>4.7.1</fhir-examples.version>
```

Comment:
if any modules rely on unreleased fhir-examples, we will need to flip this back to the latest snapshot

Reply:
In my erase work, I'll flip back to snapshot

Reply:
it turns out we already merged a commit that relied on snapshot, so i reverted this part of the change